app/scripts/modules/core/src/managed/ManagedResourceObject.tsx

```diff
@@ -25,19 +26,61 @@ function getIconTypeFromKind(kind: string) {
   return kindIconMap[getKindName(kind)] ?? 'placeholder';
 }
 
-export const ManagedResourceObject = ({
-  resource: {
+const resourceKindParsingRegex = /(?<provider>\w+)\/(?<resourceType>[\w-]+)@(?<version>\w+)/;
```

Comment:
just for education purposes (it has no behavioral implications here): the first component of the `kind` string is not actually cloud provider — that's just coincidentally what most of them use right now. That's actually a "namespace" that can be anything, roughly equivalent to NPM package namespaces.